sdk/js/src/Client.ts

```diff
@@ -50,7 +49,10 @@ export class DVCClient implements Client {
     constructor(environmentKey: string, user: DVCUser, options: DVCOptions = {}) {
         this.logger = options.logger || dvcDefaultLogger({ level: options.logLevel })
         if (options?.reactNative) {
-            this.store = new ReactNativeStore(this.logger)
+            (async () => {
```

Comment:
if this is async, won't that cause all the `this.store` references below not to work?

Reply:
for example: 
```
const storedAnonymousId = this.store.loadAnonUserId()
```